src/main/resources/twilio-csharp/options/CreateOptions.mustache

```diff
@@ -19,6 +19,11 @@
         }
     {{/vendorExtensions.x-required-param-exists}}
 
+    {{^vendorExtensions.x-is-json}}
         {{>options/GetParams}}
```

Comment:
Test cases are failing because, GetParams is commented in ReadOptions but there is ovveride keyword in 
https://github.com/twilio/twilio-csharp/blob/main/src/Twilio/Rest/Accounts/V1/Credential/AwsOptions.cs#L138
1. Remove override using twilio-oai-generator
2. Generate C# code using twilio-oai-generator and create a PR with main branch. Make sure there are no other changes then **override** removal.

Reply:
Created a PR for above mentioned:
https://github.com/twilio/twilio-csharp/pull/699
You can review that and merge after reviewing.